btsieve/tests/test_data/find_transaction_if_blockchain_reorganisation/generate_test_data.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -e
+
+source "../lib.sh"
+
+# This directory is created by docker as root
```

Comment:
haaa, that's why there is sudo everywhere?
pro-tip: create the directory yourself with mktemp instead.

Reply:
Gonna give it a go! Thanks for the suggestion :)